Makefile

```diff
@@ -33,6 +36,10 @@ build-plugins:
 	go build -o ./bin/plugins/ ./plugins/verifier/sample
 	go build -o ./bin/plugins/ ./plugins/verifier/sbom
 
+.PHONY: build-main
```

Comment:
had a discussion with @etrexel  offline, i ll work on moving controller registry under the ratify serve command instead of having a new binary